### ISSUE TITLE
Forgot to add scale arg in exp_informer

### ISSUE THREAD

**Issue by OskarEM** [zhouhaoyi/Informer2020#584]
In the exp_informer, it is forgotten to add scale as an arg. This makes it so it will always go to default init argument which is True. So when you try to take False as an argument to the informer training, then it will still be the default value of True.

**Comment by stephanielees** [comment]
I agree with this. Right now, when we run `exp.train(setting)`, the `_get_data` method from the `exp_informer` class will give us the scaled data, and no options to change it. It'll be great if we have the options to scale our data or just train the raw data.
